daemon/src/projection.rs

```diff
@@ -1214,8 +1214,11 @@ impl TryFrom<Order> for CfdOrder {
             Origin::Theirs => order.position_maker.counter_position(),
         };
 
+        // FIXME: this won't work anymore because we don't know what leverage the user wants to
```

Comment:
🙃 Should we record this as a ticket? I understand why this was added, but I am not sure it's very helpful to merge this.

Reply:
Definitely. I wasn't sure how to solve this yet. Could be a bigger task. 
Are you fine with leaving the todo and having the ticket?

Reply:
#1983 craeted